src/main/resources/templates/jenkins/c/Jenkinsfile

```diff
@@ -26,50 +26,55 @@ pipeline {
                     timestamps {
                         sh '''
                             cd $WORKSPACE
-                            echo &quot;--------------------setup-------------------&quot;
-                            echo &quot;User:&quot;
+                            echo "--------------------setup-------------------"
+                            echo "User:"
                             whoami
-                            echo &quot;Updating assignment and test-reports ownership...&quot;
-                            echo &quot;--------------------setup-------------------&quot;
-                            echo &quot;--------------------info--------------------&quot;
+                            echo "Updating assignment and test-reports ownership..."
+                            sudo chown artemis_user:artemis_user assignment/ -R
+                            sudo mkdir test-reports
+                            sudo chown artemis_user:artemis_user test-reports/ -R
+                            echo "--------------------setup-------------------"
+                            echo "--------------------info--------------------"
                             python3 --version
                             pip3 --version
                             gcc --version
                             # Generic debug infos:
                             # tree
-                            echo &quot;--------------------tests-------------------&quot;
+                            echo "--------------------info--------------------"
+                            echo "--------------------tests-------------------"
                             ls -la tests
-                            echo &quot;--------------------tests-------------------&quot;
-                            echo &quot;--------------------assignment--------------&quot;
+                            echo "--------------------tests-------------------"
+                            echo "--------------------assignment--------------"
                             ls -la assignment
```

Comment:
are these logs actually needed?

Reply:
I think it's activated for the Bamboo environment so I think it's needed

Reply:
I have talked with @krusche and removed them.